Library/Homebrew/dev-cmd/sh.rb

```diff
@@ -29,7 +29,9 @@ def run
         ENV.activate_extensions!(env: args.env)
 
         if superenv?(args.env)
-          T.cast(ENV, Superenv).deps = Formula.installed.select { |f| f.keg_only? && f.opt_prefix.directory? }
+          ENV.deps = Formula.installed.select do |f|
```

Comment:
This `cast` and the one in `--env` are no longer necessary. I broke the block into multiple lines, bc otherwise `rubocop` wants to postfix the `if` guard, which is too much for one line, IMO.